selfdrive/car/volkswagen/values.py

```diff
@@ -176,11 +177,11 @@ class VWCarInfo(CarInfo):
   CAR.AUDI_Q3_MK2: VWCarInfo("Audi Q3 2020-21", "ACC + Lane Assist"),
   CAR.SEAT_ATECA_MK1: VWCarInfo("SEAT Ateca 2018"),
   CAR.SEAT_LEON_MK3: VWCarInfo("SEAT Leon 2014-20"),
-  CAR.SKODA_KAMIQ_MK1: VWCarInfo("Škoda Kamiq 2021", footnotes=[Footnote.KAMIQ]),
-  CAR.SKODA_KAROQ_MK1: VWCarInfo("Škoda Karoq 2019"),
-  CAR.SKODA_KODIAQ_MK1: VWCarInfo("Škoda Kodiaq 2018-19"),
-  CAR.SKODA_SCALA_MK1: VWCarInfo("Škoda Scala 2020"),
-  CAR.SKODA_SUPERB_MK3: VWCarInfo("Škoda Superb 2015-18"),
+  CAR.SKODA_KAMIQ_MK1: VWCarInfo("Škoda Kamiq 2021", footnotes=[Footnote.KAMIQ, Footnote.VW_HARNESS]),
+  CAR.SKODA_KAROQ_MK1: VWCarInfo("Škoda Karoq 2019", footnotes=[Footnote.VW_HARNESS]),
+  CAR.SKODA_KODIAQ_MK1: VWCarInfo("Škoda Kodiaq 2018-19", footnotes=[Footnote.VW_HARNESS]),
+  CAR.SKODA_SCALA_MK1: VWCarInfo("Škoda Scala 2020", footnotes=[Footnote.VW_HARNESS]),
+  CAR.SKODA_SUPERB_MK3: VWCarInfo("Škoda Superb 2015-18", footnotes=[Footnote.VW_HARNESS]),
```

Comment:
I'm not sure if the footnote isn't needed on all SEAT and Audi entries as well.  Wait until they are confirmed in the wild?